zerver/lib/actions.py

```diff
@@ -4366,10 +4366,39 @@ def do_send_create_user_group_event(user_group: UserGroup, members: List[UserPro
                  )
     send_event(event, active_user_ids(user_group.realm_id))
 
-def check_add_user_group(realm: Realm, name: Text, initial_members: List[UserProfile],
+def send_user_group_update_notification(acting_user: UserProfile, group_name: Text,
+                                        members: List[UserProfile], action: Text) -> None:
+    if action == "add_members":
+        message_template = """Hi there! We thought you'd like to know that {} ({}) \
+added you to the user group **{}**."""
+    elif action == "remove_members":
+        message_template = """Hi there! We thought you'd like to know that {} ({}) \
+removed you from the user group **{}**."""
+    elif action == "delete_group":
+        message_template = """Hi there! We thought you'd like to know that {} ({}) \
+deleted the user group **{}**."""
+
+    for member in members:
+        if member != acting_user:
+            message = message_template.format(acting_user.full_name, acting_user.email, group_name)
+            internal_send_private_message(member.realm, get_system_bot(settings.NOTIFICATION_BOT),
```

Comment:
We check for if this is None in other parts of this file. Possibly that's a holdover from past, and we should just assume we always have a NOTIFICATION_BOT?